### PR TITLE
Implemeted modules

### PR DESCRIPTION
**Pull request by JOdiemo**
Use ES6 modules to write modular JavaScript.
Learn how to use proper ES6 syntax.
Understand improvements that ES6 brings to JavaScript.
Use npm as software packages system.

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+body {
+  margin: 5px;
+  padding: 5px;
+  box-sizing: border-box;
+  width: 100%;
+}
+
+body::before {
+  content: "";
```
Comment (TedLivist) [inline index.css:15]: - [ ] I am unable to click the form inputs for adding books. There is a CSS issue. Kindly see the video demo below and how to fix this.

  [screen-capture (1).webm](https://user-images.githubusercontent.com/36542986/227984578-e4306021-aab5-4b5f-bbf4-edf5ce4690c1.webm)

File: modules/applications.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
+// Form local storage availability checker function
+
+export function isStorageAvailable(type) {
```
Comment (TedLivist) [inline modules/applications.js]: - [ ] We should use ES6 syntax for our functions. Kindly refactor this to an arrow function.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @JOdiemo,

Good job so far!
✔️ Books can added
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by KayLemba — APPROVED** [review]
[[# STATUS: APPROVED :green_circle: :clap: 

Hello  @JOdiemo      , 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉  Keep up the good work 🥂 

HIGHLIGHTS:

- [x]  Your repo is professional  😎 
- [x] Descriptive P.R 💯 
- [x] Professional README 💯
- [x] Your work is well documented 🥇 
- [x] ES6 implemented🥇  
- [x] Nice and Neat UI design 💯 










 








Now you can merge this branch and move on :1st_place_medal: 
Happy Coding cheers 🥇 💯 



well-1

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._]]s

**Pull request merged**